src/utilities/geometry/FloorplanJS.hpp

```diff
@@ -191,9 +191,9 @@ class UTILITIES_API FloorplanJS
   // recursively traverses through value and finds the largest id
   void setLastId(const Json::Value& value);
 
-  Json::Value* findByHandleString(Json::Value& value, const std::string& key, const std::string& handleString);
-  Json::Value* findByName(Json::Value& value, const std::string& key, const std::string& name, bool requireEmptyHandle);
-  Json::Value* findById(Json::Value& value, const std::string& key, const std::string& id);
+  static Json::Value* findByHandleString(Json::Value& value, const std::string& key, const std::string& handleString);
+  static Json::Value* findByName(Json::Value& value, const std::string& key, const std::string& name, bool requireEmptyHandle);
+  static Json::Value* findById(Json::Value& value, const std::string& key, const std::string& id);
```

Comment:
You might think this may be a terrible idea. Because Ruby bindings.

SWIG will no longer let you do `fs = FloorPlanJS(); fs.findByName(...)`

But that's in the "private" section, so all good